ossec-server/usr/share/doc/ossec-server/changelog.Debian

```diff
@@ -1,3 +1,10 @@
+ossec-server (2.8.2) unstable; urgency=low
+
+  [ SecureDrop Team ]
```

Comment:
Both of those lines were generated using `dch -a`. I used `-a` instead of `-c` like in the securedrop/upgrade-versions.sh script to append to the changelog rather than create a new one. But the format here is the generic one generated by the dch helper script.